investmentsFintech/investmentsFintech/PINVerificationViewController.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  qViewController.swift
+//  InvestmentsFintech
+//
+//  Created by Алексей ]Чанов on 25/02/2020.
+//  Copyright © 2020 Алексей. All rights reserved.
+//
+
+import UIKit
+
+class PINVerificationViewController: UIViewController {
+    
+    // MARK: - Private properties
```

Comment:
It seems to me that is better to place all private variables and methods in one section of code. Right for now we have:
private properties
life cycle methods
private methods

And we can order sections something like this:
life cycle methods
private properties
private methods